### PR TITLE
feat: Well Markers Layer

### PR DESCRIPTION
**Pull request by LeonidPolukhin**
The layer can display markers of three flat shapes: circle, square and triangle.
For every marker the following attributes can be defined:
- center position in 3D.
- azimuth,  rotation angle relative to north direction.
- inclination, rotation angle relative to vertical direction.
- size.
- fill color with transparency.
- outline color.

### CODE REVIEW

File: typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/* eslint-disable prettier/prettier */
```
Comment (hkfb) [inline typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.stories.tsx:1]: Should try to honor prettier rules and only override in specific cases

File: typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts [hunk 2]
```diff
@@ -0,0 +1,386 @@
+/* eslint-disable prettier/prettier */
```
Comment (hkfb) [inline typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts:1]: Should try to honor prettier rules and only override in specific cases

**Review by hkfb — CHANGES_REQUESTED** [review]

File: typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts (hunk 2 above)
Comment (LeonidPolukhin) [reply typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts:1]: oops, good catch, thanks!

Comment (LeonidPolukhin) [reply typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.ts]: prettier is enabled, style fixed.

File: typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.stories.tsx (hunk 1 above)
Comment (LeonidPolukhin) [reply typescript/packages/subsurface-viewer/src/layers/well_markers/wellMarkersLayer.stories.tsx]: prettier is enabled, style fixed.

**Review by hkfb — APPROVED** [review]

**Pull request merged**